.github/workflows/workflow-repo-standards-validation.yml

```diff
@@ -0,0 +1,120 @@
+name: Reusable workflow to validate organization standards
+
+on:
+  workflow_call:
+
+jobs:
+  validate-name:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Check repository name
+      run: |
+        repo_name="${{ github.repository }}"
+        lower_repo_name="${repo_name,,}"
+        
+        if [[ "$repo_name" != "$lower_repo_name" ]]; then
+          echo "The repository name '$repo_name' should contain only lowercase letters." >&2
+          exit 1
+        elif [[ "$repo_name" == *"_"* ]]; then
+          echo "The repository name '$repo_name' should contain dashs ( - ) and not underscores ( _ )." >&2
+          exit 1
+        fi
+      if: always()
+      shell: bash
+
+    - name: Check repository visibility
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        repo_info=$(curl -H "Authorization: token $GITHUB_TOKEN" -s "https://api.github.com/repos/${{ github.repository }}")
+        repo_visibility=$(echo $repo_info | jq -r .private)
+        
+        if [[ "$repo_visibility" == "true" ]]; then
+          echo "The repository '${{ github.repository }}' is private and needs to be public". >&2
+          exit 1
+        fi
+      if: always()
+      shell: bash
+
+    - name: Check repository description
+      env:
+        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+      run: |
+        repo_info=$(curl -H "Authorization: token $GITHUB_TOKEN" -s "https://api.github.com/repos/${{ github.repository }}")
+        repo_description=$(echo $repo_info | jq -r .description)
+        
+        if [[ "$repo_description" == "null" || -z "$repo_description" ]]; then
+          echo "The repository does not have a description, please add one." >&2
+          exit 1
+        fi
+      if: always()
```

Comment:
becuase if you don't have that, the workflow stops at the first failed test and I didn't want the workflow to fail because there was no description for example and skip the other tests, it was cleaner to have a workflow that tells you all the test that failed so that you can make all the changes at once not one by one

Reply:
You may want to add a comment at the top explaining this as it's not entirely obvious

Reply:
https://github.com/ai-cfia/github-workflows/pull/9#discussion_r1350508814
sorry this comment belonged here btw

Reply:
and yes will do

Reply:
done